### PR TITLE
R4R: Fix the CLI client section of docs

### PR DESCRIPTION
**Pull request by shilei-wang**
https://github.com/irisnet/irishub/issues/1376

### CODE REVIEW

File: docs/zh/cli-client/README.md [hunk 1]
```diff
@@ -20,15 +20,15 @@
...
 | -----------------| -----  | -------- | --------------------- | ------------------------------------------------------------------- |
 | --account-number | int    | false    | 0                     | 发起交易的账户的编号 |
-| --async          | bool   | false    | false                 | 是否异步广播交易（仅当commit为false时有效） |
+| --async          | bool   | false    | false                 | 是否异步广播交易（不对交易进行任何验证，立即返回交易的hash，仅当commit为false时有效） |
 | --commit         | bool   | false    | false                 | 广播交易并等到交易被打包再返回 |
-| --chain-id       | string | true     | ""                    | Tendermint节点的`Chain ID` |
+| --chain-id       | string | true     | ""                    | tendermint节点的`Chain ID` |
 | --dry-run        | bool   | false    | false                 | 模拟执行交易，并返回消耗的`gas`。`--gas`指定的值会被忽略 |
-| --fee            | string | true     | ""                    | 交易费 |
+| --fee            | string | true     | ""                    | 交易费（指定交易费的上线） |
```
Comment (shilei-wang) [inline docs/zh/cli-client/README.md]: （指定交易费的上线）--> （指定交易费的上限）will fix in next commit

File: docs/cli-client/bank/account.md [hunk 2]
```diff
@@ -29,21 +27,20 @@ iriscli bank account [address] [flags]
 
 | Name,shorthand        | Default        | Description                                 | Required | Type   |
 | --------------------- | -------------- | ------------------------------------------- | -------- | ------ |
-| -e, --encoding string | hex            | String   Binary encoding (hex \|b64 \|btc ) | False    | String |
-| --home string         | /root/.iriscli | Directory for config and data               | False    | String |
-| -o, --output string   | text           | Output format (text \|json)                 | False    | String |
+| -e, --encoding | hex            | String   Binary encoding (hex\b64\btc ) | False    | String |
```
Comment (chengwenxi) [inline docs/cli-client/bank/account.md]: String   Binary encoding (hex\b64\btc )   ->  Binary encoding (hex\b64\btc )

File: docs/cli-client/bank/send.md [hunk 3]
```diff
@@ -2,16 +2,14 @@
 
 ## Description
 
-Sending tokens to another address. 
+Sending tokens to another address， this command includes "create/sign/broadcast" transaction.
```
Comment (chengwenxi) [inline docs/cli-client/bank/send.md]: , this command includes `create/sign/broadcast` transaction.

File: docs/cli-client/bank/sign.md [hunk 4]
```diff
@@ -7,47 +7,75 @@ Sign transactions in generated offline file. The file created with the --generat
...
+| --generate-only  | bool   | false    | false                 | Build an unsigned transaction and write it to STDOUT |
+| --help, -h       | string | false    |                       | Print help message |
+| --indent         | bool   | false    | false                 | Add indent to JSON response |
+| --json           | string | false    | false                 | Return output in json format |
+| --ledger         | bool   | false    | false                 | Use a connected Ledger device |
+| --memo           | string | false    | ""                    | Memo to send along with transaction |
+| --node           | string | false    | tcp://localhost:26657 | \<host>:\<port> to tendermint rpc interface for this chain |
+| --print-response | bool   | false    | false                 | return tx response (only works with async = false)|
+| --sequence       | int    | false    | 0                     | Sequence number to sign the tx |
+| --trust-node     | bool   | false    | true                  | Don't verify proofs for responses |
```
Comment (chengwenxi) [inline docs/cli-client/bank/sign.md]: duplicate describe  in `docs/cli-client/README.md`

File: docs/cli-client/bank/token-stats.md [hunk 5]
```diff
@@ -28,21 +28,20 @@
 
 | Name,shorthand        | Default        | Description                                 | Required | Type   |
 | --------------------- | -------------- | ------------------------------------------- | -------- | ------ |
-| -e, --encoding string | hex            | String   Binary encoding (hex \|b64 \|btc ) | False    | String |
-| --home string         | /root/.iriscli | Directory for config and data               | False    | String |
-| -o, --output string   | text           | Output format (text \|json)                 | False    | String |
+| -e, --encoding | hex            | String   Binary encoding (hex\b64\btc ) | False    | String |
+| --home         | /root/.iriscli | Directory for config and data               | False    | String |
+| -o, --output    | text           | Output format (text\json)                 | False    | String |
 | --trace               |                | Print out full stack trace on errors        | False    |        |
```
Comment (chengwenxi) [inline docs/cli-client/bank/token-stats.md]: Remove all the global flag to `docs/cli-client/README.md`.

File: docs/cli-client/keys/add.md [hunk 6]
```diff
@@ -2,7 +2,7 @@
 
 ## Description
 
-Create a new key, or import from seed
+Create a new key, or recover from seed
```
Comment (chengwenxi) [inline docs/cli-client/keys/add.md]: recover from mnemonic

File: docs/cli-client/bank/README.md [hunk 7]
```diff
@@ -1,4 +1,4 @@
-# iriscli bank
+****# iriscli bank
```
Comment (HaoyangLiu) [inline docs/cli-client/bank/README.md]: So many `*`

File: docs/zh/cli-client/tendermint/validator-set.md [hunk 8]
```diff
@@ -1,44 +1,44 @@
 # iriscli tendermint validator-set
 
 ## 描述
-根据指定高度在验证器上查询
+在给定高度获取区块的验证人集数据。如果未指定高度，则将使用最新高度作为默认高度。
```
Comment (HaoyangLiu) [inline docs/zh/cli-client/tendermint/validator-set.md]: 查询指定高度的验证人信息，包括地址公钥，投票权重以及当前出块优先级。如果未指定高度，则查询最新高度的验证人信息。

File: docs/cli-client/service/bind.md [hunk 9]
```diff
@@ -7,29 +7,28 @@ Create a new service binding
...
+| --prices              |                         | prices of binding, will contains all method                                                                                                 |          |
+| --service-name        |                         | service name                                                                                                                                 |  Yes     |
+| --usable-time         |                         | an integer represents the number of usable service invocations per 10,000                                                                       |  Yes     |
 
 ## Examples
 
 ### Add a binding to an existing service definition
-In service binding, you need to define `deposit`, minimum deposit = `price` of this service * `MinDepositMultiple` in genesis file
-
+In service binding, you need to define `deposit`,  the minimum mortgage amount of this `deposit` is `price` * `MinDepositMultiple` (defined in genesis file)
```
Comment (chengwenxi) [inline docs/cli-client/service/bind.md]: `,  ` -> `, `

File: docs/cli-client/service/refund-deposit.md [hunk 10]
```diff
@@ -7,21 +7,24 @@ Refund all deposit from a service binding
...
-| --def-chain-id        |                         | [string] the ID of the blockchain defined of the service                                                                                              |  Yes     |
-| --service-name        |                         |  [string] service name                                                                                                                                |  Yes     |
+| --def-chain-id        |                         | the ID of the blockchain defined of the service                                                                                              |  Yes     |
+| --service-name        |                         | service name                                                                                                                                |  Yes     |
 
 ## Examples
 
 ### Refund all deposit from an unavailable service binding
+
+Before refunding , you should [disable](disable.md) the service binding first.
```
Comment (chengwenxi) [inline docs/cli-client/service/refund-deposit.md]: ` , ` -> `, `

File: docs/cli-client/service/bind.md (hunk 9 above)
Comment (chengwenxi) [reply docs/cli-client/service/bind.md]: defined by system parameters, can be modified through governance

File: docs/cli-client/bank/send.md (hunk 3 above)
Comment (shilei-wang) [reply docs/cli-client/bank/send.md]: fixed

File: docs/cli-client/bank/sign.md (hunk 4 above)
Comment (shilei-wang) [reply docs/cli-client/bank/sign.md]: fixed

File: docs/cli-client/bank/token-stats.md (hunk 5 above)
Comment (shilei-wang) [reply docs/cli-client/bank/token-stats.md]: fixed

File: docs/cli-client/keys/add.md (hunk 6 above)
Comment (shilei-wang) [reply docs/cli-client/keys/add.md]: fixed

File: docs/cli-client/bank/README.md (hunk 7 above)
Comment (shilei-wang) [reply docs/cli-client/bank/README.md]: fixed

File: docs/zh/cli-client/tendermint/validator-set.md (hunk 8 above)
Comment (shilei-wang) [reply docs/zh/cli-client/tendermint/validator-set.md]: fixed

File: docs/cli-client/service/bind.md (hunk 9 above)
Comment (shilei-wang) [reply docs/cli-client/service/bind.md]: fixed

File: docs/cli-client/service/refund-deposit.md (hunk 10 above)
Comment (shilei-wang) [reply docs/cli-client/service/refund-deposit.md]: fixed

**Pull request merged**